toolchain/defs.bzl

```diff
@@ -174,11 +175,13 @@ def _zig_repository_impl(repository_ctx):
     compile_cmd = [
         _paths_join("..", "zig"),
         "build-exe",
+        "-target",
+        _TARGET_MCPU[host_platform][0],
+        "-mcpu={}".format(_TARGET_MCPU[host_platform][1]),
         "-fstrip",
-        "-mcpu={}".format(_MCPU[host_platform]),
         "-OReleaseSafe",
         "zig-wrapper.zig",
-    ] + (["-static"] if os == "linux" else [])
+    ]
```

Comment:
I assume that `-static` for linux got dropped on purpose?

Reply:
Yes, musl impliea that.